pycbc/population/__init__.py

```diff
@@ -1,2 +1,4 @@
 from pycbc.population.rates_functions import *
 from pycbc.population.scale_injections import *
+from pycbc.population.fgmc_functions import *
```

Comment:
It would be good if you do this to explicitly have an __all__ in the modules you import from to control what actually gets up-imported (instead of *everything*)